src/screenControl.py

```diff
@@ -375,6 +380,8 @@ def processInput(self, key):
             # before exiting the program
             self.getFilesToUse()
             sys.exit(0)
+        elif self.mode == X_MODE and key in lbls:
```

Comment:
we could probably handle this case first and keep the rest of the if/else statement free from checking X_MODE. more future proof probably